year-pages/2019-conferences-list.md

```diff
@@ -2,4 +2,5 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
-| [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools. | [Yes](https://www.rubyconfindia.org/scholarship/)
\ No newline at end of file
+| [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools. | [Yes](https://www.rubyconfindia.org/scholarship/) |
+| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA TRADE TOWER, PUNE | The second Pune edition of India’s premier JavaScript conference | No
```

Comment:
Can you update the location to `MCCIA Trade Tower, Pune` and add a full-stop/period at the end of the sentence for description? Thanks!

Reply:
Hi @nikhita, I have updated the PR. Please have a look.